unit_tests/test_tester.py

```diff
@@ -0,0 +1,247 @@
+from time import sleep
+import os
+import shutil
+import logging
+
+from sdcm.tester import ClusterTester, silence, TestResultEvent
+from sdcm.sct_config import SCTConfiguration
+from sdcm.utils.log import MultilineMessagesFormatter
+from sdcm.utils.common import generate_random_string
+
+
+class FakeSCTConfiguration(SCTConfiguration):
+    def _load_environment_variables(self):
+        return {
+            'config_files': ['test-cases/PR-provision-test-docker.yaml'],
+            'cluster_backend': 'docker',
+            'store_results_in_elasticsearch': False
+        }
+
+
+class ClusterTesterForTests(ClusterTester):
+    _sct_log = None
+    _final_event = None
+    _get_event_summary_cached = None
+    _get_events_grouped_by_category_cached = None
+
+    def __init__(self, *args):
+        self.logdir = os.path.join('/tmp', generate_random_string(10))
+        os.mkdir(self.logdir)
+        logging.config.dictConfig({
```

Comment:
generalize `sdcm.utils.log.configure_logging` and use it here instead of a copy-paste